geonode/maps/templates/maps/map_detail.html

```diff
@@ -207,7 +207,7 @@ <h4 class="list-group-item-heading">{% trans "Map Layers" %}</h4>
         <li class="list-group-item">
           <h4>{% trans "Copy this map" %}</h4>
           <p>{% trans "Duplicate this map and modify it for your own purposes" %}</p>
-          <a href="{% url "new_map" %}?copy={{ resource.id }}" class="btn btn-default btn-md btn-block">{% trans "Create a New Map" %}</a>
+          <a href="{% url "new_map" %}/copy/{{ resource.id }}" class="btn btn-default btn-md btn-block">{% trans "Create a New Map" %}</a>
```

Comment:
May be it's better to resolve the url like this:
```{% url "copy_map" resource.id %}```
and set the name to `copy_map` for the url.